.gitignore

```diff
@@ -1,5 +1,6 @@
 
 charts/keycloak/realms/client_secret_518334922510-ucfeovt7mnc8felk8r6cu90kngu8tm19.apps.googleusercontent.com.json
 *.tgz
+!common-1.17.1.tgz
```

Comment:
please have a look still at the helm dependencies or see whether we can get rid of the commons at the first place

Reply:
Okay, I will investigate it